test/vagrant-service-manager/binary_handlers/adb_kubernetes_binary_handler_test.rb

```diff
@@ -0,0 +1,103 @@
+require_relative '../../test_helper'
+require 'vagrant/util/downloader'
+
+# Tests through ADBKubernetesBinaryHandler to BinaryHandler
+module VagrantPlugins
+  module ServiceManager
+    describe ADBKubernetesBinaryHandler do
+      before do
+        @machine = fake_machine
+        @options = { type: :kubernetes, '--cli-version' => '1.2.0' } # 1.2.0 is available in CDK/ADB VM
+        @base_download_url = 'https://storage.googleapis.com/kubernetes-release/release'
```

Comment:
Same here, should we have these urls all collected in a single configuration point?